app/libs/hashids.py

```diff
@@ -13,7 +13,7 @@
 def parse_hash(hashid=int, env=current_app.config['ENV']):
     decoded = str(hashids.decode(hashid)[0])
 
-    if env == 'development':
+    if env == 'development' or env == 'test':
         regex = re.compile('([0-9]{8})([0-9]{4})')
         parts = regex.match(decoded).groups()
         return 'T'.join(parts)
```

Comment:
The addition of T in a personal number should be handled in it's own function. Since it don't have anything with the actual parsing of the hash to do?

Maybe have one function to parse the hash and the one function to add the T.

So what i suggest is something in the following lines:

parsedHashId = parse_hash(hash)
if (env === "development" or env === "test")
    id = insertLetter(value=parsedHash, letter="T", position=7)

Reply:
Just a thought though, might not be relevant for this PR

Reply:
Thanks for the good input.